polls/tests.py

```diff
@@ -1,3 +1,40 @@
-from django.test import TestCase
+from django.contrib.auth import get_user_model
+from django.urls import reverse
+from django.utils import timezone
 
-# Create your tests here.
+from rest_framework import status
+from rest_framework.test import APITestCase
+
+from .models import Batch, Task, Annotation
+
+
+class LockUserAnnotationListTest(APITestCase):
+    def setUp(self):
+        self.user = get_user_model().objects.create(
+            username="test_user", password="testpass"
+        )
+        self.admin_user = get_user_model().objects.create(
+            username="test_admin", password="testpass", is_superuser=True, is_staff=True
+        )
+        self.batch = Batch.objects.create(created_at=timezone.now(), notes="test note")
+        self.task_1 = Task.objects.create(
+            batch=self.batch,
+            reference_url="http://www.test.com",
+            transform_url="http://www.example.com",
+            transform_metadata={"test": 1},
+        )
+        self.annotatation = Annotation.objects.create(
+            user=self.user,
+            task=self.task_1,
+            annotated_at=timezone.now(),
+            task_presentation="AAB",
+            annotations="XXY",
+        )
+
+    def test_lock_user(self):
+        url = reverse("lock-users-api")
+        self.client.force_authenticate(user=self.admin_user)
+        payload = {"users": [self.user.id, 5, 10]}
```

Comment:
Please use the user id dynamically.

Reply:
This is testing page and the id supplied is mainly to support the assert call, It's better off as static ids